app/services/symbolService.js

```diff
@@ -174,6 +230,6 @@ class SymbolService {
   }
 
 }
-  
+
 module.exports = new SymbolService();
-  
\ No newline at end of file
+  
```

Comment:
Why is there a blank line in the last line?

Reply:
lol, deleted.